gcp/opsverse-gke/vars.tfvars

```diff
@@ -1,10 +1,20 @@
-gcp_project_id      = "opsverse-dev-poc"
-gcp_region          = "us-east4"
-cluster_name        = "opsverse-gke-cluster"
+gcp_project_id = "opsverse-dev-poc"
+gcp_region     = "us-east4"
+cluster_name   = "opsverse-gke-cluster"
+# The bucket name should be of the format devopsnow-loki-<customer_name>-<stack_name>
+#If customer name is kylaq , the stack name is decided as alpha then bucket_name = "devopsnow-loki-kylaq-alpha" 
+bucket_name = "devopsnow-loki-<customer_name>-<stack_name>"
+
+
+#The customer_stack should be of the format <customer_name>-<stack_name> 
```

Comment:
All this information has to be added to `README.md`. Customers refer Readme file. Let's add all these details.

Reply:
Done